bin/tkldev-changelog-extractor

```diff
@@ -2,29 +2,46 @@
 from argparse import ArgumentParser
 from subprocess import check_output
 from os.path import isfile
+import os
 import sys
 import re
+import logging
 
-CHANGE_HEAD=r'^turnkey-.*-(\d+)\.(\d+)-?(?:rc|beta)? \((\d+)\) .*$'
+CHANGE_HEAD=r'^turnkey-[a-z0-9-]+-(\d+)\.(\d+)-?(?:rc|beta)?\d? \((\d+)\) .*$'
 CHANGE_FOOTER=r' -- .*'
 
+if 'DEVDEBUG' in os.environ:
+    log_level = logging.DEBUG
+else:
+    log_level = logging.CRITICAL
+logging.basicConfig(level=log_level,
+                    format='%(asctime)s %(name)-4s %(levelname)-5s %(message)s',
+                    datefmt='%H:%M:%S',
+                    )
+logger = logging.getLogger()
```

Comment:
We should name this logger, in case other loggers are used during the execution of tkldev-detective.

something like 

```python3
logger = logging.getLogger("tkldev-detective.changelog-extractor")
```

Reply:
done: https://github.com/turnkeylinux/tkldev-detective/pull/4/commits/5486a1529933323e91f71aa261cc39b40c6b6f79